### PR TITLE
feat: added rating component

### PR DESCRIPTION
**Pull request by dev-prabhat**
image

image

### CODE REVIEW

File: component.css [hunk 1]
```diff
@@ -24,6 +26,7 @@
   --warning-light-color: #fef9c3;
   --light-color: #cbd5e1;
   --dark-color: #1e293b;
+  --rating-color: orange;
```
Comment (zoef-yt) [inline component.css]: stay consistent and give this in hex too

File: components/rating.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  position: relative;
+}
+/* hide the inputs */
+.star-rating input {
+  display: none;
+}
+/* set properties of all labels */
+.star-rating > label {
+  width: 30px;
+  height: 30px;
```
Comment (zoef-yt) [inline components/rating.css]: use rem

File: components/rating.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  font-family: Arial;
+  font-size: 30px;
```
Comment (zoef-yt) [inline components/rating.css]: use rem

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by zoef-yt — APPROVED** [review]
Good code. Nice mention of changes in the description. good to merge.

**Review by sach10-create — APPROVED** [review]
Good to merge

**Review by zoyatanweer — APPROVED** [review]
good to merge with suggested changes

**Review by sumitk-singh — APPROVED** [review]
code looks clean, good to merge

**Pull request merged**